pytr/event.py

```diff
@@ -55,15 +56,19 @@ class EventType(Enum):
     "INTEREST_PAYOUT": PPEventType.INTEREST,
     "INTEREST_PAYOUT_CREATED": PPEventType.INTEREST,
     # Removals
+    "OUTGOING_TRANSFER": PPEventType.REMOVAL,
     "OUTGOING_TRANSFER_DELEGATION": PPEventType.REMOVAL,
     "PAYMENT_OUTBOUND": PPEventType.REMOVAL,
     "card_order_billed": PPEventType.REMOVAL,
     "card_successful_atm_withdrawal": PPEventType.REMOVAL,
     "card_successful_transaction": PPEventType.REMOVAL,
     # Saveback
     "benefits_saveback_execution": ConditionalEventType.SAVEBACK,
+    # Sparechange
+    "benefits_spare_change_execution": ConditionalEventType.SPARECHANGE,
```

Comment:
Any reason to create a new ConditionalEventType? 
```suggestion
    "benefits_spare_change_execution": ConditionalEventType.TRADE_INVOICE,
```

Reply:
Done and tested. Works.